src/utils/beforeBlockString.js

```diff
@@ -10,24 +11,26 @@
  * @param {boolean} [options.noRawBefore] - Leave out the `before` string
  * @return {string}
  */
-export default function (statement, { noRawBefore } = {}) {
-  let result = ""
-  if (statement.type !== "rule" && statement.type !== "atrule") { return result }
+export default function (
+  statement: Object,
+  { noRawBefore }: { noRawBefore: ?boolean} = {}
+): string {
+  let result: string = ""
```

Comment:
This is ok, but a little excessive. No need to annotate variables, unless they are of a special type.